src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,9 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String typeBulldozer;
```

Comment:
add a constructor with parameters to initialize these values. applicable for all entities